src/averageAges.js

```diff
@@ -1,5 +1,23 @@
 'use strict';
 
+function getChildren(who, arr) {
+  return who.filter(
+    person => arr.some(one => one.name === person.mother)
+  );
+}
+
+function getAge({ died, born }) {
+  return died - born;
+}
+
+function getSex(arr, letter) {
+  return arr.filter(person => person.sex === letter);
+}
+
+function findMothers(people, womenArray) {
+  return people.find(person => person.mother === womenArray.name);
+}
```

Comment:
```suggestion
function findMother(people, woman) {
  return people.find(person => person.mother === woman.name);
}
```

Reply:
what does `find` return? a single item or a list of them? that's why you should name functions appropriately